anime-list-master.xml

```diff
@@ -28554,8 +28554,7 @@
   <anime anidbid="10384" tvdbid="79214" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Mushishi Zoku Shou</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;1-2;2-3;</mapping>
-      <mapping anidbseason="1" tvdbseason="0">;11-4+5;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;11-2;</mapping>
```

Comment:
I don't see s00e03 anywhere else, wouldn't it be part of this one as `;11-2+3;`?

Reply:
So I saw that Alrefai did that in their pr, but look at the episode length. AniDB E11 is 50 minutes long, basically a "double-length" episode. TVDB S00E02 is 47 minutes long, also a "double-length" episode (and would be rounded to 50m per AniDB's rounding scheme). So it's clear that those are meant to be the same thing. I think S00E03 (which is 25 minutes long) only exists to serve as the second part of the two-parter episode; but S00E02 contains both parts already. S00E03 probably isn't supposed to exist in the same TVDB "order" as S00E02, they usually put split multi-part episodes into a different viewing order in their database.

Reply:
Maybe it's worth adjusting the duration on TVDB for "Part 1" then? Feels like it would be better than that orphaned episode leaving a hole in people's libraries 🤔 

The split-view for specials on TVDB is fairly recent (I personally hate it) and episodes need to be tagged manually as belonging to a category (and I haven't in my recent edits, because I hate that unordered view and want it to die in a fire xD), so that doesn't tell much.

Reply:
Er, by TVDB "order" I mean Airing Order vs DVD Order vs. other alt orders.
Adjusting S00E02 duration is probably not the desired process by TheTVDB. Going by their own standards, this special was originally aired as an hour-long special, so only the 47 minutes episode (S00E02) should exist under Aired Order/Specials. Since this special was later released on physical media apparently as two individual episodes, a second episode entry was created to be inserted for DVD Order S02E12. Aired Order S00E02 was reused as DVD Order S02E11, but that would be incorrect since DVD Order S02E11 is also supposed to be 25 minutes long, being equal to only the first half of the aired special.
What should probably happen is:
1. Another 25 minute episode for the first half be created and assigned to DVD Order S02E11, and move the current DVD Order S02E11 to DVD Order "Unassigned Episodes"
2. Aired Order S00E03 moved to Aired Order "Unassigned Episodes"
Unfortunately I cannot perform this second action with just edits, I will need to open a ticket to get these edits. That should also get mod attention and if they decide the solution is to actually modify Aired Order S00E02 duration, then that would be official. Why the hell did they close down the forum...
See [Ascension](https://thetvdb.com/series/ascension) for a similar scenario where they use two-parter and single-episode equivalents for different Orders like this.

Reply:
I see, sorry for the misunderstanding. Seems I feel more strongly than I was aware about those new segmented views 😅 

Oh well, it's a problem of syncing of the 2 DBs moderation then, out of the scope of this list so it seems sensible to merge this as-is. It should be a simple commit if TVDB ever figures this situation out in a way that doesn't leave a hole in the form of a "half-duplicate" ep03

Reply:
Haha, I actually like that they are starting to finally standardize organization of specials... before, you couldn't even renumber/change ordering so it never made much sense, so organizing by category seems a step in the right direction.
And yeah, I figure there will be another commit anyway because if they choose option one, TVDB S00E04 and S00E05 will probably decrement in number and need to be remapped; and if they choose option two then we'd change remapping to be ;11-2+3; as you suggested. But for now with their database "unclean" this is the best mapping I could come up with.

Reply:
I hate the new views precisely because they take episodes out of order visually. You can have s00e05 showing up above s00e03 if they're in different categories, absolute madness!